daemon/info.go

```diff
@@ -111,3 +112,13 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 
 	return v, nil
 }
+
+// The uppercase and the lowercase are available for the proxy settings.
+// See the Go specification for details on these variables. https://golang.org/pkg/net/http/
+func getProxyEnv(key string) string {
+	proxyValue := os.Getenv(strings.ToUpper(key))
+	if proxyValue == "" {
+		return os.Getenv(key)
```

Comment:
That doesn't mention anything about it. It also wouldn't belong there as the behavior is in regards to the usage of the `getProxyEnv` function, which a user of the CLI has no interaction with.